### DISCUSSION TITLE
Private repositories option grayed out under Member privileges in my organization

### DISCUSSION THREAD

**Discussion by chargerback-ron** [community/community#23702] [Enterprise]
<p>Under Memeber Privileges for my organization, in the grouping Repository Creation, the option for Private repositories is grayed out.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/e3ac1fd46dd38b64f247be4047904ef649669ed0.png" data-download-href="/uploads/short-url/wu59WSHdOZ3vdMo4pUi6IOyrPZS.png?dl=1" title="GitHub Settings Issue Private.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10093772/181078817-3f2988a3-0f8a-4213-a705-f08d66f1c717.png" alt="GitHub Settings Issue Private.png" data-base62-sha1="wu59WSHdOZ3vdMo4pUi6IOyrPZS" width="690" height="393" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/e3ac1fd46dd38b64f247be4047904ef649669ed0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">GitHub Settings Issue Private.png</span><span class="informations">1107×632 34.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I wish to enable this option so that only Private repos can be created. My organization is already marked private. We have a 5 seat paid account, which shows under billing that we can create unlimited private repositories.</p>

**Comment by chargerback-ron** [comment, marked as answer]
<p>Okay, I have my answer. In case anyone else comes across this, it is an Enterprise Cloud feature, and all I have is Team access.</p>

**Comment by elucidsoft** [comment]
<p>This is a ridiculous restriction.  We are already paying for Private repos, and all I want to do is  make sure one of my contractors doesn’t accidentally mark a repo as public.  </p>

**Comment by BrentFarese** [comment]
<p>I agree this is ridiculous.  If you are paying for private repos, you should be able to ensure no one makes a mistake and changes a repo to be public!  This is not an “enterprise” feature, it is a normal feature.</p>

**Comment by unitof** [comment]
<p><a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/restricting-repository-creation-in-your-organization" rel="nofollow noopener">Source</a> for the policy:</p>
<blockquote>
<p>To allow members to create private repositories only, your organization must use GitHub Enterprise Cloud. For more information, see “<a href="https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-repository-visibility" rel="nofollow noopener">About repository visibility</a>.”</p>
</blockquote>
<p>I agree this seems an excessive restriction, and common use case for non-enterprise users to not want to allow their employees/contractors to inadvertantly make a repo public and leak data/code.</p>
<p>What is the best forum to respectfully request this feature be unlocked for GitHub Team customers?</p>

**Comment by kenorb** [comment]
<p>It’s not Enterprise feature, you can change it in your <em>Organization settings</em> at /organizations/[ORG]/settings/member_privileges</p>
<blockquote>
<p>Repository forking: Allow forking of private repositories</p>
</blockquote>
<blockquote>
<p>If enabled, forking is allowed on private and public repositories. If disabled, forking is only allowed on public repositories. This setting is also configurable per-repository.</p>
</blockquote>

**Comment by chargerback-ron** [comment]
<p>Maybe I am missing something. But what does forking have to do with new repository creation?</p>
